paasta_tools/cli/schemas/kubernetes_schema.json

```diff
@@ -190,7 +190,124 @@
                     "type": "string"
                 },
                 "autoscaling": {
-                    "type": "object"
+                    "type": "object",
+                    "properties": {
+                        "metrics_provider": {
+                            "enum": [
+                                "uwsgi",
+                                "mesos_cpu",
+                                "cpu",
+                                "piscina",
+                                "http",
```

Comment:
if i remove this setting from compute-infra-test-service, how would you feel about dropping this as an allowed provider?

Reply:
yeah let's do it

Reply:
coolio, just made an internal PR for this :)